src/plotman/manager.py

```diff
@@ -81,9 +95,9 @@ def maybe_start_new_plot(dir_cfg: plotman.configuration.Directories, sched_cfg:
     youngest_job_age = min(jobs, key=job.Job.get_time_wall).get_time_wall() if jobs else MAX_AGE
     global_stagger = int(sched_cfg.global_stagger_m * MIN)
     if (youngest_job_age < global_stagger):
-        wait_reason = 'stagger (%ds/%ds)' % (youngest_job_age, global_stagger)
+        wait_reason = 'global stagger (%ds/%ds)' % (youngest_job_age, global_stagger)
```

Comment:
If there's something to clarify wouldn't it be that this is for the time-stagger not the phase-stagger?  Though the reporting of seconds adjacent to the `stagger` comment it seems to lead that way.

Reply:
It seems that the only relevant need for the seconds to be displayed is for the global stagger m setting.
To me, it looks poor, having a clock that goes passed it's target and keeps ticking when it is no longer relevant.
Maybe keep showing youngest_job_age and remove global_stagger, once global_stagger has passed?
I had thought about adding an option of putting individual running time counters (hh:mm) in front of each running plot so people would know their overall duration, but as a different PR.

Reply:
It lets you know how far past the time stagger you have gone based on some other restriction.  Going beyond the time stagger isn't inherently bad. 
 I specifically set my time stagger a bit less than even spacing would be based on my total jobs and job duration.  Anyways, to satisfy "everyone" I expect I'll need to make most of the pieces optional.  There's a PR adding a couple lines and another adding a couple columns and...  `:]`  And I don't super want to do that in curses when I've already started a switch to prompt_toolkit.

Reply:
all wait_reason defines returned to original